simulariumio/tests/converters/test_trajectory_converter.py

```diff
@@ -59,1011 +62,1180 @@ def mixed_agents_wrong_display_type2():
-                                0.0,
-                                1001.0,
-                                3.0,
-                                1.0,
-                                0.0,
-                                0.0,
-                                0.0,
-                                0.0,
-                                0.0,
-                                0.0,
-                                1.0,
-                                6.0,
-                                7.77508859,
-                                260.16762947,
-                                -171.02427873,
-                                -20.46326319,
-                                179.43194042,
-                                485.07810635,
-                                1000.0,
-                                400.0,
-                                1.0,
-                                7.77508859,
-                                260.16762947,
-                                -171.02427873,
-                                0.0,
-                                0.0,
-                                0.0,
-                                0.5,
-                                0.0,
-                            ],
-                        },
-                    ],
+                "lookAtPosition": {
+                    "x": DEFAULT_CAMERA_SETTINGS.LOOK_AT_POSITION[0],
+                    "y": DEFAULT_CAMERA_SETTINGS.LOOK_AT_POSITION[1],
+                    "z": DEFAULT_CAMERA_SETTINGS.LOOK_AT_POSITION[2],
                 },
-                "plotData": {
-                    "version": CURRENT_VERSION.PLOT_DATA,
-                    "data": ["plot data goes here"],
+                "upVector": {
+                    "x": DEFAULT_CAMERA_SETTINGS.UP_VECTOR[0],
+                    "y": DEFAULT_CAMERA_SETTINGS.UP_VECTOR[1],
+                    "z": DEFAULT_CAMERA_SETTINGS.UP_VECTOR[2],
                 },
+                "fovDegrees": DEFAULT_CAMERA_SETTINGS.FOV_DEGREES,
+            },
+        ),
+    ],
+)
+def test_camera_defaults(camera, expected_camera):
+    assert camera == expected_camera
+
+
+# test type mapping
+@pytest.mark.parametrize(
+    "typeMapping, expected_typeMapping",
+    [
+        (
+            default_agents_data["trajectoryInfo"]["typeMapping"],
+            default_agents_type_mapping(),
+        ),
+        (
+            mixed_agents_data["trajectoryInfo"]["typeMapping"],
+            mixed_agents_type_mapping(),
+        ),
+        (
+            fiber_agents_data["trajectoryInfo"]["typeMapping"],
+            fiber_agents_type_mapping(),
+        ),
+    ],
+)
+def test_type_mapping(typeMapping, expected_typeMapping):
+    assert typeMapping == expected_typeMapping
+
+
+# test model info
+@pytest.mark.parametrize(
+    "modelInfo, expected_modelInfo",
+    [
+        (
+            mixed_agents_data["trajectoryInfo"]["modelInfo"],
+            {
+                "title": "Some agent-based model",
```

Comment:
same issue here as I wrote above: it's important to test the case where you send in nothing here and make sure it doesn't break